pallets/tendermint-light-client/src/types.rs

```diff
@@ -522,6 +620,45 @@ impl TryFrom<HeaderStorage> for Header {
     }
 }
 
+impl From<Header> for HeaderStorage {
```

Comment:
same here, `impl TryFrom`

Reply:
I know, it's just Tendermint's usage of `Hash` and `Option` is crap and can't even be mapped correctly... And now I have to turn all `From`s into `TryFrom`s...

Reply:
my IDE at least can do it automagically..